portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+
+<body>
+    <!-- Navigation -->
+	<nav>
+		<ul>
+		    <li><a href="index.html">Home</a></li>
```

Comment:
When you get to the group project, you'll want to keep this formatted to avoid conflicts.  Your editor can do that for you with Alt+Shift+I